sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -196,10 +196,28 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
             raise ValueError("No token scope provided.")
         return _generate_sas_token(scopes[0], self.policy, self.key)
 
+class AzureSasTokenCredential(object):
```

Comment:
Let's give this a more specific name to avoid any developer confusion between this and the Core credential object.
Something like `ServiceBusSasToken`

Reply:
renamed to ServiceBusAzureSasTokenCredential (ServiceBusSasTokenCredential is already taken)